Common/include/CConfig.hpp

```diff
@@ -3728,6 +3798,29 @@ class CConfig {
    */
     su2double GetThermal_Conductivity_ConstantND(void) const { return Thermal_Conductivity_ConstantND; }
 
+  /*!
+   * \brief Get the value of the constant mass diffusivity for scalar transport.
+   * \return Constant mass diffusivity.
+   */
+  su2double GetDiffusivity_Constant(void) const { return Diffusivity_Constant; }
+
+  /*!
+   * \brief Get the value of the non-dimensional constant mass diffusivity.
+   * \return Non-dimensional constant mass diffusivity.
+   */
+  su2double GetDiffusivity_ConstantND(void) const { return Diffusivity_ConstantND; }
```

Comment:
Especially for mass diffusivity there is prob some overlap with NEMO so i have to take a look at that